packages/nextcloud/lib/src/helpers/core.dart

```diff
@@ -0,0 +1,22 @@
+// ignore_for_file: public_member_api_docs
+
+enum ShareType {
+  user(0),
+  group(1),
+  usergroup(2),
+  link(3),
+  email(4),
+  // 5 was contact, is no longer used
```

Comment:
I'd rather have the deprecated ones be implemented with a `deprecated` annotation.

That way `ShareType.values[index]` will always match the requested index.